Add Short Rest! to OBR Store

ManuelLovell opened the pull request:
Make sure that your submission has the following:

- [x] A link to the raw github markdown file with a YAML front matter definition containing:
  - [x] The title of your extension
  - [x] A description of your extension
  - [x] The author
  - [x] A hero image
  - [x] An icon
  - [x] Tags (these will help with your extension's discoverability)
  - [x] A link to your extensions manifest

Please, go through these steps before you submit a PR.

- [x] You have done your changes in a separate branch.

- [x] You have a descriptive commit message with a short title (first line).

- [x] You have only one commit (if not, squash them into one commit).

- [x] Your pull request MUST target the `main` branch on this repository.

- [x] Your pull request puts your extensions details as the last entry in the extensions.json file

nthouliss commented:
Thanks for your patience. The only feedback I have would to be to add some responsiveness on the GM role when revoked. The extension won't respond to GM role changes (if I add or revoke) until a page refresh. 

Not sure if it was just me but being able to stop playing tetris after starting it would be neat.

ManuelLovell commented:
> Thanks for your patience. The only feedback I have would to be to add some responsiveness on the GM role when revoked. The extension won't respond to GM role changes (if I add or revoke) until a page refresh.
> 
> Not sure if it was just me but being able to stop playing tetris after starting it would be neat.

Thanks for checking in;
I've added an end-game button and a button-legend for the game.
I also fixed the role handling.
